src/accelerate/test_utils/scripts/external_deps/test_zero3_integration.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch.distributed
+from transformers import AutoModel, TrainingArguments
```

Comment:
Accelerate doesn't rely on `transformers` so we need an import check for `is_transformers_available` here, and a `@requires_huggingface_suite` in the  function itself

Reply:
added